pkg/pool-cow/test/foundry/CowRouter.t.sol

```diff
@@ -781,7 +1179,11 @@ contract CowRouterTest is BaseCowTest {
         // Test vault balances
         assertEq(
             balancesAfter.vaultTokens[daiIdx],
-            balancesBefore.vaultTokens[daiIdx] + donations[daiIdx] - expectedProtocolFees[daiIdx] + daiSwapAmountIn,
+            balancesBefore.vaultTokens[daiIdx] +
+                donations[daiIdx] -
+                expectedProtocolFees[daiIdx] +
+                daiSwapAmountIn +
+                lostDaiTokens,
```

Comment:
Maybe a small explanation, such as:
- Tokens going into the vault:
  - Donation
  - Amount in
  - Amounts locked in case amounts transferred were lower than the hint
- Tokens going out of the vault:
  - Protocol fees (collected by router)